Reescrevendo a documentação da PrestaShop (#1408)
mercadopago/devsite-docs: guides/plugins/prestashop/introduction.en.md

# PrestaShop
 

> [PrestaShop](https://www.prestashop.com/en/) is an e-commerce platform with open source software that allows any user to create and develop a commercial website. As an official PrestaShop partners, we provide greater security and efficiency to your store.

Reviewer: ```suggestion
[PrestaShop](https://www.prestashop.com/en/) is an e-commerce platform with open source software that allows any user to create and develop a commercial website. As official PrestaShop partners, we provide greater security and efficiency to your store.
```

> This documentation was designed so that you can do all the necessary configuration to integrate the Mercado Pago plugin with your store on PrestaShop, without having to write a single line of code to install it in your customers' stores.

Reviewer: ```suggestion
This documentation was designed so that you can perform all the necessary configuration steps to integrate the Mercado Pago plugin with your store on PrestaShop, without having to write a single line of code to install it in your customers' stores.
```

 
## Integration features
 

> With the Mercado Pago plugin you can configure the payment using [Checkout Pro](https://www.mercadopago.[FAKER][URL][DOMAIN]/developers/en/guides/online-payments/checkout-pro/introduction), where payment is made on the Mercado Pago page (inside or outside your store) with cards (debit or credit) and cash (balance account in Mercado Pago and boleto).

Reviewer: ```suggestion
With the Mercado Pago plugin you can configure the payment using [Checkout Pro](https://www.mercadopago[FAKER][URL][DOMAIN]/developers/en/guides/online-payments/checkout-pro/introduction), where payment is made on the Mercado Pago page (inside or outside your store) with cards (debit or credit) and cash (balance account in Mercado Pago and boleto).
```

> There is also possibility to use the [Custom Checkout](https://www.mercadopago.[FAKER][URL][DOMAIN]/developers/en/guides/online-payments/checkout-api/introduction), in which the payment it is carried out by credit card without leaving your virtual store and the **Ticket Checkout**, with payments being made in person with cash or boleto.

Reviewer: ```suggestion
There is also possibility to use the [Custom Checkout](https://www.mercadopago[FAKER][URL][DOMAIN]/developers/en/guides/online-payments/checkout-api/introduction), in which the payment it is carried out by credit card without leaving your virtual store and the **Ticket Checkout**, with payments being made in person with cash or boleto.
```

> There is also possibility to use the [Custom Checkout](https://www.mercadopago.[FAKER][URL][DOMAIN]/developers/en/guides/online-payments/checkout-api/introduction), in which the payment it is carried out by credit card without leaving your virtual store and the **Ticket Checkout**, with payments being made in person with cash or boleto.

Reviewer: Custom checkout: no deberia ser "Checkout >Transparente" para Brasil y "Checkout API" para el resto de hispanos?

> Another feature of the integration is the possibility of activating binary mode, a feature that allows you to configure the automatic approval or rejection of payments. In addition, users will also be able to pay in your store without being obligatorily registered with Mercado Pago.

Reviewer: ```suggestion
Another feature of the integration is the possibility to activate binary mode, a feature that allows you to configure the automatic approval or rejection of payments. In addition, users will also be able to pay in your store without being obligatorily registered with Mercado Pago.
```

>  

Reviewer: Si el título es "Important" no debería ser un warning en lugar de un note?
